Frameworks/QuartzCore/CATransform3D.mm

```diff
@@ -314,9 +314,9 @@ CATransform3D CATransform3DTranslate(CATransform3D cur, float x, float y, float
  @Status Interoperable
 */
 CATransform3D CATransform3DConcat(CATransform3D a, CATransform3D b) {
```

Comment:
There are no unit tests for these (although these should be testable).  Since you have validated this, would you please add a unit test for the methods you touched.  You would have to add a new unit test project for QuartzCore.